packages/hop-node/src/db/TransfersDb.ts

```diff
@@ -311,6 +311,19 @@ class TransfersDb extends TimestampedKeysDb<Transfer> {
     })
   }
 
+  async getTransfersWithTransferRootHash (transferRootHash: string) {
+    await this.tilReady()
+    const transfers = await this.getTransfersFromWeek()
```

Comment:
This part can be improved by indexing transfer ids by transfer root hash/id but want to get more feedback first on the overall PR

Reply:
No because it'll only lookup data where it matches it transfer root hash (or id). The key will be something like `${transferRootHash}:${transferId}`

Reply:
Updated with https://github.com/hop-protocol/hop/pull/323/commits/21e23682c9f8788e7960cbac209fe113218f4518